addon/utils.js

```diff
@@ -1,6 +1,9 @@
 import _ from 'lodash'
 import Ember from 'ember'
-const {A} = Ember
+const {A, get} = Ember
+import {getModelPath} from 'bunsen-core/utils'
+
+const assign = Ember.assign || Object.assign || Ember.merge
```

Comment:
Odd. Is this to support old versions of Ember? Also: Is there any reason to expect that `Object.assign` will not be available?